Example Apps/Example Swift/ProxyManager.swift

```diff
@@ -149,8 +151,10 @@ private extension ProxyManager {
 // MARK: - SDLManagerDelegate
 
 extension ProxyManager: SDLManagerDelegate {
-    /// Called when the connection beween this app and SDL Core has closed.
+    /// Called when the connection beween this app and the module has closed.
     func managerDidDisconnect() {
+        subscribeButtonManager.unsubscribeToPresetButtons()
```

Comment:
This shouldn't be necessary because it's disconnected. The manager should clear its subscriptions when a disconnect occurs.

Reply:
Fixed. Removed